5.2.cup_of_join.py

```diff
@@ -0,0 +1,19 @@
+# The function accepts an unlimited number of lists and an option for an additional parameter,
+# and concatenates them into one list and the additional parameter between them
+def join(*lists, sep='-'):
+    if not lists:  # If no lists were received at all
+        return None
+    lst = [sep]
+    res = []
+    # Concatenate the lists into one list with the 'sep' parameter between them
+    for list1 in lists:
+        res += list1 + lst
+    res.pop()
+    # print(res)
+    return res
+
+
+join([1, 2], [8], [9, 5, 6], sep='@')
```

Comment:
Use name==main -
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/

Reply:
I fixed it.